woocommerce/woocommerce-android
# [Manual Taxes M#1] Tax rates education dialogDisplay tax lines and "tax based on" setting in order creation

#### This PR adds displaying of tax lines and "tax based on" setting in order creation screen.

Closes: #9625


⚠️ Do not merge before corresponding PR in FluxC is merged and FluxC version is updated in this PR: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2812

Note - there are the following follow-up PRs:
- https://github.com/woocommerce/woocommerce-android/pull/9651
- https://github.com/woocommerce/woocommerce-android/pull/9647

### Description

This PR displays active tax rate information - tax lines from the Order model with the "Tax based on" setting from `wp-admin > WooCommerce > Settings > Tax`.

### Testing instructions

You'll need to test with the following FluxC branch:  https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2812 or review and merge FluxC PR first.

Verify you have tax rates defined under `wp-admin > WooCommerce > Settings > Tax > Standard Rates`
Verify your tax-based-on setting under `wp-admin > WooCommerce > Settings > Tax`
1. Create new order
2. Add product
3. Observe tax lines being displayed in the payments section (tax rate name + "tax based on" value)
Try the above with the modified settings.

### Images/gif

<img width=300 src=https://github.com/woocommerce/woocommerce-android/assets/4527432/ef79deef-ee9d-432e-986d-dda99e3b1d9d/>

I'll update notes in the next PR: tax rates education dialog.
